### DISCUSSION TITLE
Is there a way to remove the year detail from the date picker?

### DISCUSSION THREAD

**Discussion by DrSharky** [wojtekmaj/react-date-picker#570] [Q&A] [answered]
I looked over the props, but I'm a bit confused on whether this is possible to do. I would like to use the date picker, but I don't want to have the year selectable, just month and days.
I suppose for now I can configure the maxDate prop to be the end of the current year, and the minDate to be the beginning of the current year. Then after clicking a date, I can just delete the year value. Let me know if there is a simpler way to do this. Thanks!

**Comment by wojtekmaj** [comment, marked as answer]
Unfortunately that's not possible - without year Date Picker would produce invalid Date. Disabling input should be easy, but the remaining problem would be the Calendar.
